source/support.tex

```diff
@@ -4043,6 +4053,7 @@
 \indexlibraryglobal{end}%
 
 \begin{codeblock}
+// all freestanding
```

Comment:
Pre-existing, but this goes immediately into the class synopsis. It needs its own, actual header synopsis, like `<source_locator>`'s.

Reply:
I think this is beyond the scope of this paper, no?

Reply:
Opened #5720 for that.

Reply:
thanks!